### PR TITLE
fix(meeting): join with recording notification

### PR DESCRIPTION
**Pull request by chrisadubois**
fixes https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-98302

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ import Parser from '../locus-info/parser';
 export default class LocusInfo extends EventsScope {
   constructor(updateMeeting, locus = {}, webex, meetingId) {
     super();
+    this.updateControls(locus.controls);
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: Could we move this to ~line 58 with the other `update` functions to keep them grouped together? If not, let's add a comment as to why this is inconsistent.

Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: we cannot, so i will add a comment

Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: i'll replace this.controls with this.updateControls

**Comment by adamweeks** [comment]
Looks like this needs a rebase, sorry @chrisadubois

**Comment by chrisadubois** [comment]
we are putting this in a larger bug fix PR as one.

**Pull request closed without merging**